src/sourcetransform.rs

```diff
@@ -1,18 +1,23 @@
-use crate::error::Error::SourceTransformerError;
-use crate::error::ErrorKind::UserDefinedError;
-use crate::shared::{self, prost_timestamp_from_utc};
+use crate::error::Error::{self, SourceTransformerError};
```

Comment:
order imports? is there a rustfmt option for this?

Reply:
There's an option [group_imports](https://rust-lang.github.io/rustfmt/?version=v1.4.38&search=#group_imports)
But it's nightly. I will follow it's `StdExternalCrate` import order manually here.